.betterer.results

```diff
@@ -8716,6 +8729,35 @@ exports[`better eslint`] = {
     "public/app/plugins/panel/geomap/editor/MapViewEditor.tsx:5381": [
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"]
     ],
+    "public/app/plugins/panel/geomap/editor/StyleEditor.tsx:5381": [
```

Comment:
I don't get why these are all new -- this moved a file to a different folder 🤷‍♂️